api/schema.py

```diff
@@ -0,0 +1,28 @@
+from drf_spectacular.openapi import AutoSchema
+from rest_framework.permissions import SAFE_METHODS
+
+from api.serializers.errors import ValidationErrorSerializer
+
+
+class CustomSchema(AutoSchema):
+    """
+    Differences from default schema:
+    - Default schema uses same tag (from url) for all endpoints.
+      To make it more clear, we use viewset name as additional tag.
```

Comment:
OK great. Maybe just a small example of how the url will be different from default.
Maybe it's clear for guys with lots of API experience, just not for a noob like me :)

Reply:
Added